src/rest_framework_reactive/consumers.py

```diff
@@ -6,12 +6,49 @@
 from channels.consumer import AsyncConsumer, SyncConsumer
 from channels.generic.websocket import JsonWebsocketConsumer
 
+from django.core.cache import cache
+
+from .connection import get_queryobserver_settings
 from .models import Observer, Subscriber
 from .observer import QueryObserver
 from .protocol import *
 
 # Maximum number of cached observer executors.
 MAX_CACHED_EXECUTORS = 1024
+# Waiting observers cache name
+WAITING_OBSERVERS_CACHE_PREFIX = 'waiting_observer_'
```

Comment:
It is a good practice to use per app name-spacing for cache keys, e.g. `drf_reactive_waiting_observer_`.